### ISSUE TITLE
how to original webpage all css and js files were from different folder hierarchy Dowlaod

### ISSUE THREAD

**Issue by ghost** [PramodKhare/GetMeThatPage#2]
hello Dear 
 I had apply those code but still only one html page download 

How to original web page all css and js files were from different folder hierarchy

have you any idea Please help me

**Comment by PramodKhare** [comment]
Do you mean - you want to download a single webpage and keep the folder hierarchy of css and js files?

You can certainly do that, you will have to add another method which does the folder hierarchy creation.

Let me know if that's what you are trying to do or something else.

**Comment by ghost** [comment]
yes I used GrabWebPage class class but still only one page download index.html 
but I want to all folder and page I want  to all page and folder download (like,Image folder css folder all )

**Comment by ghost** [comment]
hello dear 
 have you demo for folder hierarchy Dowlaod website,plz give me.
